### PR TITLE
Fix golint for pkg/mflag

### PR DESCRIPTION
**Pull request by fcantournet**
Addresses (partly) : https://github.com/docker/docker/issues/14756

Signed-off-by: Félix Cantournet felix.cantournet@cloudwatt.com

### CODE REVIEW

File: pkg/mflag/flag.go [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ func (f *FlagSet) Parse(arguments []string) error {
 // incorrect number of arguments. It returns error only if error handling is
 // set to ContinueOnError and parsing fails. If error handling is set to
 // ExitOnError, it's safe to ignore the return value.
-func (cmd *FlagSet) ParseFlags(args []string, withHelp bool) error {
+func (f *FlagSet) ParseFlags(args []string, withHelp bool) error {
```
Comment (runcom) [inline pkg/mflag/flag.go]: `fs` :yum: 
you also changed the method receiver name and tests are failing for this

Comment (fcantournet) [inline pkg/mflag/flag.go]: ok will change it. (and ty silly me)
I'll add pkg/mflag to the automatically linted packges too.

Comment (calavera) [inline pkg/mflag/flag.go]: there is a compilation issue because `cmd` is still referenced in line 1077.

**Comment by fcantournet** [comment]
@runcom does this LGTY ?

**Comment by calavera** [comment]
Please, do NOT merge this PR just yet. I'd like to land #13771 first.

**Comment by calavera** [comment]
Nothing bad about the changes, but we need the other PR merged as soon as possible and I'd like to avoid any friction before that happens.

**Comment by calavera** [comment]
@fcantournet we merged #13771 yesterday and this is ready to merge. Unfortunately, we added a couple new methods to the mflag package in master. Do you want to rebase this change and cover the new additions?

**Comment by fcantournet** [comment]
@calavera Sure, I'll rebase and finish the job.

**Comment by calavera** [comment]
LGTM

File: pkg/mflag/flag.go [hunk 2]
```diff
@@ -420,7 +420,7 @@ func Lookup(name string) *Flag {
 	return CommandLine.formal[name]
 }
 
-// Indicates whether the specified flag was specified at all on the cmd line
+// IsSet Indicates whether the specified flag was specified at all on the cmd line
```
Comment (vdemeester) [inline pkg/mflag/flag.go]: s/Indicates/indicates/ and a period at the end :blush:

**Comment by fcantournet** [comment]
@calavera test seem flaky. The errors don't seem to be link with this patch. is there a way to retrigger the checks ?

**Comment by RichardScothern** [comment]
LGTM

**Comment by dmcgowan** [comment]
LGTM

**Comment by LK4D4** [comment]
Ok, it compiles
LGTM

**Pull request merged**